background.js

```diff
@@ -16,6 +16,40 @@ function tabCallback(tabId, changeInfo, updatedTab) {
   }
 }
 
+function handleMaybeInt(maybeString) {
+  const maybeInt = parseInt(maybeString, 10);
+  if (Number.isNaN(maybeInt)) {
+    return null;
+  }
+  return maybeInt;
+}
+
+function handlePassphraseCleaning() {
+  chrome.storage.local.get(["lastModified", "lockAfterSeconds", "temporaryPassphrase"], function(data) {
+    const currentTime = Date.now();
+    const lastModified = handleMaybeInt(data['lastModified']);
```

Comment:
nit: `data['lastModified']` can be written `data.lastModified`, same for other results.